mm2src/mm2_main/tests/integration_tests_common/mod.rs

```diff
@@ -12,26 +11,18 @@ use mm2_test_helpers::structs::{CreateNewAccountStatus, HDAccountAddressId, HDAc
 use serde_json::{self as json, Value as Json};
 use std::collections::HashMap;
 use std::env::var;
-#[cfg(target_arch = "wasm32")] use std::str::FromStr;
 
 /// This is not a separate test but a helper used by `MarketMakerIt` to run the MarketMaker from the test binary.
 #[test]
 #[cfg(not(target_arch = "wasm32"))]
 fn test_mm_start() { test_mm_start_impl(); }
 
-pub fn test_mm_start_impl() {
+#[cfg(not(target_arch = "wasm32"))]
+fn test_mm_start_impl() {
     if let Ok(conf) = var("_MM2_TEST_CONF") {
```

Comment:
nit: we can merge these 2 funcs. as of now, doesn't seem that this indirection is useful.